step-plans/step-plans-core/src/main/java/step/core/artefacts/reports/ReportNodeAccessor.java

```diff
@@ -36,9 +37,9 @@ public interface ReportNodeAccessor extends Accessor<ReportNode>, ReportTreeAcce
 
 	Iterator<ReportNode> getChildren(ObjectId parentID, int skip, int limit);
 
-	Iterator<ReportNode> getReportNodesByExecutionID(String executionID);
+	Stream<ReportNode> getReportNodesByExecutionID(String executionID);

```

Comment:
Add comment mentioning that the stream has to be closed

Reply:
javadoc added